src/components/facet-header.tsx

```diff
@@ -0,0 +1,88 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_faceting.scss';
+
+import { useRef, useState } from 'react';
+
+// Components
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+import { Displayname } from '@isrd-isi-edu/chaise/src/models/displayname';
+
+type FacetHeaderProps = {
+  displayname: Displayname; // content to be displayed as panel header
```

Comment:
This is not what I meant. Do it like you did in the other PR in the jsdoc style. The goal is to make sure vscode (or any other tools) can find the comment and show it.

```
  /**
   * comment goes here
   */
  displayname: Displayname;
```